src/test/java/org/padaiyal/utilities/commandline/StreamUtilityTest.java

```diff
@@ -0,0 +1,35 @@
+package org.padaiyal.utilities.commandline;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests for StreamUtility.
+ */
+class StreamUtilityTest {
+
+  /**
+   * Tests StreamUtility::convertInputStreamToString() with valid input.
+   *
+   * @throws IOException If there is an issue converting the input stream to a string.
+   */
+  @Test
+  public void testConvertInputStreamToString() throws IOException {
+    String string = "ZXCVBNM!@#$%";
```

Comment:
add the string to a test.properties file

Reply:
Has been converted to a parameterized test instead.